official/README.md

```diff
@@ -108,44 +135,6 @@ os.environ['PYTHONPATH'] += ":/path/to/models"
 pip3 install --user -r official/requirements.txt
 ```
 
----
-
-## More models to come!
-
-The team is actively developing new models.
-In the near future, we will add:
-
-- State-of-the-art language understanding models:
-  More members in Transformer family
-- Start-of-the-art image classification models:
-  EfficientNet, MnasNet and variants.
-- A set of excellent objection detection models.
-
-If you would like to make any fixes or improvements to the models, please
-[submit a pull request](https://github.com/tensorflow/models/compare).
-
----
-
 ## Contributions
 
-Every model should follow our guidelines to uphold our objectives of readable,
-usable, and maintainable code.
-
-### General Guidelines
-
-- Code should be well documented and tested.
-- Runnable from a blank environment with ease.
-- Trainable on: single GPU/CPU (baseline), multiple GPUs & TPUs
-- Compatible with Python 3 (using [six](https://pythonhosted.org/six/)
-when being compatible with Python 2 is necessary)
-- Conform to
-  [Google Python Style Guide](https://github.com/google/styleguide/blob/gh-pages/pyguide.md)
-
-### Implementation Guidelines
-
-These guidelines are to ensure consistent model implementations for
-better readability and maintainability.
-
-- Use [common utility functions](utils)
-- Export SavedModel at the end of the training.
-- Consistent flags and flag-parsing library ([read more here](utils/flags/guidelines.md))
```

Comment:
nit: Do you feel the absolute path would be better?

Reply:
GitHub recommends using relative links to refer to other files within the repository.